### PR TITLE
altered migration for games and added piece model migration

### PR DESCRIPTION
**Pull request by evanpineo**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,34 +10,49 @@
...
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
+
+  create_table "games", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+    t.integer  "game_id"
```
Comment (mscoutermarsh) [inline db/schema.rb]: We won't need a `game_id` inside of `game`. Rails auto adds an `id` column for all tables.

We do need a `game_id` inside of Pieces though. 😄

File: db/migrate/20171107235209_create_pieces.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.string :color
+      t.string :type
+      t.integer :x_position
+      t.integer :y_position
+      t.timestamps
+    end
```
Comment (mscoutermarsh) [inline db/migrate/20171107235209_create_pieces.rb:9]: Needs `game_id`

And `belongs_to :game` in the model.

File: db/schema.rb (hunk 1 above)
Comment (gamalhetari) [reply db/schema.rb]: We did not add the `game_id` in games that was already there.

File: db/migrate/20171107235209_create_pieces.rb (hunk 2 above)
Comment (gamalhetari) [reply db/migrate/20171107235209_create_pieces.rb:9]: We will add the `game id` in piece migration asap. We already added `belongs_to :game` in the piece model.

**Pull request merged**